src/main/java/seedu/address/model/delivery/Delivery.java

```diff
@@ -85,7 +85,7 @@ public boolean equals(Object other) {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append("Deliverer's Name")
+        builder.append("Deliverer's Name: ")
```

Comment:
Can we follow what's on master's branch for this part? Deliverer's name is not a suitable term